webapp/src/pages/Settings.tsx

```diff
@@ -362,6 +381,67 @@ const Settings: React.FC<Props> = ({ open, onClose }) => {
       ),
     });
 
+  const displayPipelineAddSection = (pipelineIndex: number) => (
+    <Divider
+      sx={{
+        "&::before": { width: (theme) => theme.spacing(2) },
+        [`& .${dividerClasses.wrapper}`]: { padding: 0 },
+      }}
+    >
+      <Tooltip title="Add pipeline">
+        <IconButton
+          sx={{ color: (theme) => theme.palette.grey[400] }}
+          aria-label="add-pipeline"
+          disabled={isUpdatingConfig}
+          onClick={() =>
+            updatePartialConfig({
+              pipelines: [
+                ...resultingConfig.pipelines!.slice(0, pipelineIndex + 1),
+                ...defaultConfig.pipelines!,
+                ...resultingConfig.pipelines!.slice(pipelineIndex + 1),
+              ],
+            })
+          }
+        >
+          <AddCircle />
+        </IconButton>
+      </Tooltip>
+    </Divider>
+  );
+
+  const displayPostprocessorAddSection = (
+    pipelineIndex: number,
+    postprocessorIndex: number,
+    postprocessor: PipelineDefinition["postprocessors"]
+  ) => (
+    <Divider
+      textAlign="left"
+      sx={{
+        "&::before": { width: (theme) => theme.spacing(2) },
+        [`& .${dividerClasses.wrapper}`]: { padding: 0 },
+      }}
+    >
+      <Tooltip title="Add postprocessor">
+        <IconButton
+          sx={{ color: (theme) => theme.palette.grey[400] }}
+          aria-label="add-postprocessor"
+          disabled={isUpdatingConfig}
+          onClick={() =>
+            updatePipeline(pipelineIndex, {
+              postprocessors: [
+                ...postprocessor!.slice(0, postprocessorIndex + 1),
+                ...defaultConfig.pipelines![0].postprocessors!,
```

Comment:
It doesn't make sense to add multiple times the same postprocessor class (in this case `Threshold`), so I would suggest adding an empty one and letting the user choose the class.
```suggestion
                { class_name: "", args: [], kwargs: {}, remote: null },
```